x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/detection_rules_client/converters/convert_rule_response_to_alerting_rule.ts

```diff
@@ -37,10 +40,17 @@ type RuntimeFields =
   | 'execution_summary';
 
 export const convertRuleResponseToAlertingRule = (
+  actionsClient: ActionsClient,
```

Comment:
nit: here again, I would either have the `actionsClient` as the second argument, or have one object argument

Reply:
Fixed here: 851e19a7a790070bd3e19de2d278627c52a2933d